### PR TITLE
Create install-ubuntu-2004.rst

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

**Comment by fakela** [comment]
> Awesome job, @fakela! This is great. Looking forward to reviewing it. :)

Thank you @justinegeffen

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (svelle) [inline source/install/install-ubuntu-2004-postgresql.rst:24]:
```suggestion
`` sudo -u postgres psql``
```

Maybe combine step two and three into this single step for simplicity?

**Review by svelle — COMMENTED** [review]
Just one small nitpick, other than that it looks good to me. Thanks @fakela :)

**Comment by fakela** [comment]
Thanks @svelle for the suggestion

File: source/install/install-ubuntu-2004-mysql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-mysql.rst:20]:
```suggestion
4. Log in to MySQL as *root*.
```

File: source/install/install-ubuntu-2004-mysql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-mysql.rst:50]:
```suggestion
    If you have installed MySQL on its own server, you need to edit the ``/etc/mysql/mysql.conf.d/mysqld.cnf`` file and comment out the ``bind-address = 127.0.0.1`` using the ``#`` symbol. Then restart your SQL server.
```

File: source/install/install-ubuntu-2004-mysql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-mysql.rst:18]: I think we can remove this as it's referenced in the preceding point.

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:24]: This may need to be right-indented.

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:49]:
```suggestion
9. (Optional) If you use a different server for your database and the Mattermost app server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/10/main/postgresql.conf`` in a text editor as *root* user. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Comment (justinegeffen) [reply source/install/install-ubuntu-2004-postgresql.rst:49]: I'm 4/5 for removing "app" from "Mattermost app server". @fakela, did you notice that in any of the existing documentation?

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:67]:
```suggestion
    a. Open ``/etc/postgresql/10/main/pg_hba.conf`` in a text editor as *root* user.
```

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:77]:
```suggestion
  **If the Mattermost server and the database are on different machines:**
```

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:79]:
```suggestion
    a. Open ``/etc/postgresql/10/main/pg_hba.conf`` in a text editor as *root* user.
```

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:81]:
```suggestion
    b. Add the following line to the end of the file, where ``{mattermost-server-IP}`` is the IP address of the Mattermost server.
```

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:102]:
```suggestion
  The PostgreSQL interactive terminal starts. To exit the PostgreSQL interactive terminal, type ``\q`` and press ENTER
```

File: source/install/install-ubuntu-2004.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004.rst:7]:
```suggestion
Install a production-ready Mattermost system on up to three machines.
```

Comment (justinegeffen) [reply source/install/install-ubuntu-2004.rst:7]: Out of scope of this PR: I'm not sure whether "machines" is a good term to use. I am 3/5 for using "servers". @jasonblais, thoughts?

File: source/install/install-ubuntu-2004-mysql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-mysql.rst:24]:
```suggestion
5. Create the Mattermost user *mmuser*.
```

Comment (justinegeffen) [reply source/install/install-ubuntu-2004-mysql.rst:24]: Note: This is a change that needs to be made across all the install guides.

File: source/install/install-ubuntu-2004-mysql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-mysql.rst:36]:
```suggestion
7. Grant access privileges to the user *mmuser*.
```

File: source/install/install-ubuntu-2004-mysql.rst
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-mysql.rst:51]: These additional lines can be deleted.

File: source/install/install-ubuntu-2004.rst
Comment (jasonblais) [reply source/install/install-ubuntu-2004.rst:7]: I'm 0/5 as well, but is the terminology used in other install guides.

Comment (justinegeffen) [reply source/install/install-ubuntu-2004.rst:7]: I think best to leave as-is for now then. I'll note it down as something to consider adjusting in future. Thanks!

**Review by justinegeffen — APPROVED** [review]
Barring the suggestions made, I am happy with this. Thanks, @fakela, this is great work and excellent timing.

**Comment by justinegeffen** [comment]
Marked as "do not merge" as I'm waiting for final approval on https://github.com/mattermost/docs/pull/3848 which includes changes to the navigation. Once approved, I'll add this guide to the navigation and link to it where referenced.

File: source/install/install-ubuntu-2004-postgresql.rst
Comment (fakela) [reply source/install/install-ubuntu-2004-postgresql.rst:49]: Same here, lemme make the needed change

File: source/install/install-ubuntu-2004-mysql.rst
Comment (fakela) [reply source/install/install-ubuntu-2004-mysql.rst:22]: > Note: This is a change that needs to be made across all the install guides

Noted 👍

File: source/install/install-ubuntu-2004.rst
Comment (jasonblais) [inline source/install/install-ubuntu-2004.rst:18]: Is the Ubuntu-1804-mattermost guide the same as for Ubuntu 20.04?

Comment (fakela) [reply source/install/install-ubuntu-2004.rst:18]: @jasonblais I checked two different articles before starting so yeah they are quite similar

**Comment by justinegeffen** [comment]
> @justinegeffen did we run through the steps to verify it works as expected?

That was my final step and I was going to check with @svelle whether he would be able to give it a quick test prior to merge. :)

**Review by svelle — APPROVED** [review]
tested and working on a ec2 ubuntu 20.04 instance

**Pull request merged**
